app/meta/src/main/resources/application.yml

```diff
@@ -21,23 +21,26 @@ server:
 # See https://docs.spring.io/spring-boot/docs/current/reference/html/production-ready-monitoring.html
 # For details
 management:
-  port: 8080
-  security:
-    enabled: true
+  server:
+    port: 8181
```

Comment:
This port differs from the `managment.port: 8080` above, is that expected?

Reply:
I have noticed that the liveness checks have ever been using port 8181 (https://github.com/syndesisio/syndesis/blob/master/install/operator/pkg/generator/assets/infrastructure/04-syndesis-meta.yml.tmpl#L104) I do not know why the local setting was using 8080 as management port but I think it makes sense to align with what we use on OCP in the config maps.